app/components/ucsf/ClinicianActions/ClinicianActions.module.scss

```diff
@@ -0,0 +1,50 @@
+@import '~styles/utils/_helpers.scss';
+
+.clinicianActions {
+  position: relative;
```

Comment:
This is already being set by the base Modal, right? Could we get rid of this? I think it's in general tricky to keep track of relative positioning in CSS when split across a base CSS class and an extended one, and it'll be easier to maintain if only the base CSS class is in charge of both this element's relative positioning as well as anything else that relies on that relative positioning.

I don't see you relying on relative positioning for any of the styles in this file, since I don't see anything with an explicit top/right/bottom/left attribute, z-index, or any of the obvious ones, so I think this component can be agnostic of the relativeness of the parent modal.

Reply:
Nope, no need! I think it was left over before I switched over to using the parent component.